docs/content/sips/010-key-value.md

```diff
@@ -0,0 +1,60 @@
+title = "SIP 010 - Key-Value Storage"
+template = "main"
+date = "2023-01-23T01:01:01Z"
+---
+
```

Comment:
Should we add a "Future Support" section that discusses what other KV interfaces we could support (and not support in the future). For example, we may support `get-many` and `set-many` functions but not the atomic `increment` function?